src/App.php

```diff
@@ -112,6 +112,15 @@ class App
     /** @var array<string, View> Modal view that need to be rendered using JSON output. */
     private $portals = [];
 
+    /**
+     * @var string used in method App::url to build the url
```

Comment:
rm the type incl. the comment where used

for type, use strong type, same for `urlBuildingExt` prop

Reply:
done